cmd/fs_mgmt/src/fs_mgmt.c

```diff
@@ -20,6 +20,7 @@
 #include <limits.h>
 #include <string.h>
 #include "cborattr/cborattr.h"
+#include "tinycbor/cbor_cnt_writer.h"
```

Comment:
This include isn't used by mynewt, I would suggest you to make it conditional.

Reply:
ok